staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -687,6 +689,29 @@ message LinuxContainerConfig {
     LinuxContainerSecurityContext security_context = 2;
 }
 
+// WindowsSandboxSecurityContext holds platform-specific configurations that will be 
+// applied to a sandbox.
+// These settings will only apply to the sandbox container.
+message WindowsSandboxSecurityContext {
+    // User name to run the container process as. If specified, the user MUST
+    // exist in the container image and be resolved there by the runtime;
+    // otherwise, the runtime MUST return error.
+    string run_as_username = 1;
+
+    // The contents of the GMSA credential spec to use to run this container.
```

Comment:
nit: is this a json "as is"? Sometimes there is a confusion when json used whether wrapping braces are included or not.

Reply:
This was copied over from the ContainerSecurityContext: https://github.com/kubernetes/kubernetes/blob/bc637a7a07cf917140daa3bd434eac9bbd8fe92c/staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto#L697-L704 

The json used here is passed directly through to the runtime as is and I believe it includes the brakets.  The way this is used in practice is via the gmsa webhook: https://github.com/kubernetes-sigs/windows-gmsa/ which converts a CRD to the json.

Reply:
resolved